src/containers/MyNdla/components/UserInfo.tsx

```diff
@@ -44,86 +42,24 @@ export const UserInfo = ({ user }: Props) => {
       }
       <ShortInfoDiv>
         <Text element="p" textStyle="content-alt" margin="none">
-          {t('user.username')}: {user.uid}
+          {t('user.username')}: {user?.username}
         </Text>
         <Text element="p" textStyle="content-alt" margin="none">
-          {t('user.name')}: {user.displayName}
+          {t('user.name')}: {user?.displayName}
         </Text>
         <Text element="p" textStyle="content-alt" margin="none">
-          {t('user.mail')}: {user.mail?.join(', ')}
+          {t('user.mail')}: {user?.username}
         </Text>
-        {user.preferredLanguage && (
-          <Text element="p" textStyle="content-alt" margin="none">
-            {t('user.preferredLanguage')}:{' '}
-            {t(`languages.${user.preferredLanguage}`)}
-          </Text>
-        )}
-        {user.mobile && (
-          <Text element="p" textStyle="content-alt" margin="none">
-            {t('user.mobile')}: {user.mobile}
-          </Text>
-        )}
       </ShortInfoDiv>
       <UnOrderedList>
-        {parsedUser.organizations.map((org) => (
+        {user?.groups.map((org) => (
           <Text element="li" textStyle="content-alt" margin="none" key={org.id}>
             {`${org.displayName}${
-              org.membership.primarySchool
-                ? ` (${t('user.primarySchool')})`
-                : ''
+              org.isPrimarySchool ? ` (${t('user.primarySchool')})` : ''
             }`}
-            <UnOrderedList>
-              {Object.entries(org.children).map(([groupType, val]) => {
-                if (val.length < 1) return null;
-                return (
-                  <Text
-                    element="li"
-                    textStyle="content-alt"
-                    margin="none"
-                    key={groupType}
-                  >
-                    {t(`user.groupTypes.${groupType}`)}
-                    <UnOrderedList>
-                      {val.map((group) => (
-                        <Text
-                          element="li"
-                          textStyle="content-alt"
-                          margin="none"
-                          key={group.id}
-                        >{`${group.displayName}${
-                          group.grep ? ` (${group.grep.code})` : ''
-                        }`}</Text>
-                      ))}
-                    </UnOrderedList>
-                  </Text>
-                );
-              })}
-            </UnOrderedList>
```

Comment:
Blir uansett ikkje brukt til nokke, og strengt tatt så henter vi ikkje lenger dette fra feide heller.

Reply:
Tenker eg må revurdere dette litt, for vi henter jo alle grupper og returnerer, men no berre med navn. Så enten filtrere slik at vi kun henter gitte typer grupper eller så vise fram dettte igjen.

Reply:
Endra i backend til å kun ta vare på fc:group grupper.